.circleci/config.yml

```diff
@@ -271,7 +271,7 @@ jobs:
           password: $DOCKERHUB_PASSWORD
     steps:
       - checkout
-      - run: cat docs/source/release_notes.rst | grep ":pr:\`${CIRCLE_PULL_REQUEST##https://github.com/FeatureLabs/featuretools/pull/}\`"
+      - run: cat docs/source/release_notes.rst | grep ":pr:\`${CIRCLE_PULL_REQUEST##https://github.com/alteryx/featuretools/pull/}\`"
```

Comment:
Not sure if you want to fix other instances of this as well:
- https://github.com/alteryx/featuretools/search?q=https%3A%2F%2Fgithub.com%2FFeatureLabs

Reply:
I'll make an issue for it, we can do a separate PR